.github/workflows/cpdb_build.yml

```diff
@@ -1,19 +1,37 @@
-name: z(TODO) CPDB - 🏗️ BUILD
+name: CPDB - 🏗️ BUILD
 
-on: workflow_dispatch
+on: 
+  workflow_call:
+    inputs:
+      run_export:
+        type: boolean
+        required: true
+  workflow_dispatch:
+    inputs:
+      run_export:
+        description: "Run Export Step and Upload to DO"
+        type: boolean
+        required: true
+        default: false
```

Comment:
is it worth adding CPDB to `build.yml` and dropping the `workflow_dispatch` trigger here?